src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -356,6 +371,7 @@ def load_prevalence_from_incidence_and_duration(key: str, location: str) -> pd.D
 
 
 def load_remission_rate_from_duration(key: str, location: str) -> pd.DataFrame:
```

Comment:
Nit: maybe add something about how NN has been zeroed out in the function name in case we try to repurpose this method somewhere else later.